lib/tasks/data_migrations/migrate_contact_metadata_to_native_fields.rb

```diff
@@ -0,0 +1,42 @@
+namespace :data_migrations do
+  task migrate_contact_metadata_to_native_fields: :environment do
+    Account.find_each do |account|
+      puts "Migrating contacts for account: #{account.id}"
+
+      account.contacts.find_each do |contact|
+        ApplicationRecord.transaction do
+          contact.iso_country_code = PhonyRails.country_from_number(contact.msisdn)
+          contact.language_code = contact.metadata["language_code"]
+          contact.save!
+
+          # EWS Cambodia
+          contact.metadata.fetch("commune_ids", []).each do | commune_id |
+            commune = Pumi::Commune.find_by_id(phone_call_metadata(:commune_code))
+
+            contact.addresses.find_or_create_by!(
+              iso_region_code: commune.province.iso3166_2,
```

Comment:
I remember we discussed about denormalizing `iso_country_code` in the addresses table, but I can't remember did we decide to do it?

Reply:
Yes, we discussed and just leave them for now, but I couldn't remember 100% as well 😂